modules/op-using-pipelines-as-code-with-a-github-app.adoc

```diff
@@ -1,105 +1,17 @@
 // This module is included in the following assemblies:
 // * pac/using-pipelines-as-code-repos.adoc
 
-:_mod-docs-content-type: PROCEDURE
+:_mod-docs-content-type: CONCEPT
 [id="using-pipelines-as-code-with-a-github-app_{context}"]
-= Using {pac} with a GitHub App 
+= Using {pac} with a GitHub App
 
 [role="_abstract"]
-GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to {pipelines-shortname}. Cluster administrators can configure a single GitHub App for all cluster users. For GitHub Apps to work with {pac}, ensure that the webhook of the GitHub App points to the {pac} event listener route (or ingress endpoint) that listens for GitHub events.
+GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to {pipelines-shortname}. Cluster administrators can configure a single GitHub App for all cluster users. For GitHub Apps to work with {pac}, ensure that the webhook of the GitHub App points to the {pac} controller route (or ingress endpoint) that listens for GitHub events.
 
-[NOTE]
-====
-When importing an application using *Import from Git* and the Git repository has a `.tekton` directory, you can configure `pipelines-as-code` for your application.
-====
+There are three ways to set up a GitHub app for {pac}:
 
+* Use the `tkn` command line utility
+* Use the Administrator perspective of the web console
+* Set up the app manually in GitHub and then create a secret for {pac}
```

Comment:
These sentences should probably have periods at the end even if they're part of a list:
```suggestion
* Use the `tkn` command line utility.
* Use the Administrator perspective of the web console.
* Set up the app manually in GitHub and then create a secret for {pac}.
```